.github/workflows/track-shipping-version.yml

```
@@ -0,0 +1,34 @@
+on:
+  release:
+    types: [released]
+
+permissions:
+  contents: write
+
+env:
+  shipping_branch_prefix: 'shipping'
```

nit: 
```suggestion
  shipping_branch_prefix: 'shipped'
```

"shipping" makes it sound like it the shipping procedure is currently active whereas "shipped" indicates that the shipping procedure is complete